src/ram/AbstractOperator.h

```diff
@@ -65,6 +56,10 @@ class AbstractOperator : public Expression {
         return equal_targets(arguments, other.arguments);
     }
 
+    NodeVec getChildNodesImpl() const override {
```

Comment:
Please rename to getChildren() etc.; the Impl suffix is very hard to read for people joining the project. 
This is not a drama because these methods are only used for traversing the tree (visit/apply/etc). It is better to keep the AST readable.

Reply:
This is done to match AST, which also uses the `getChildNodesImpl` convention to return a `vector<Node const*>` internal structure. The mutable/non-mutable overloads are handled in the base `{ast,ram}::Node` class. Can't rename it `getChildNodes` since it'd be an overload that only differs by return type. Could change it so that it subsumes the const-ptr-return case (and match both AST/RAM). Let me know your preferences.